docs/scripts_md/seriesuid2fileid.md

```diff
@@ -0,0 +1,42 @@
+# NAME
+
+seriesuid2fileid -- a script that displays a report about the pipeline insertion
+progress and outcome (including MRI violation status) of imaging datasets, based
+on series UID(s) provided as STDIN.
+
+# SYNOPSIS
+
+perl tools/seriesuid2fileid
+
+There are no available options. Once the script is invoked, series UID can be
+input to display the status. The $profile file for database connection
```

Comment:
either replace $profile by profile or put $profile in code format

Reply:
ok